README.md

```diff
@@ -7,11 +7,60 @@
 
 ## Overview
 
-This lesson is a very high-level over view of the React component lifecycle. We
-want to walk students through the whole shebang.
+As you've learned previously, a React component has two sets of properties: **props** and **state**. Props are given to the component by it's parent. You can think of props as an external influence that the component has no control over - like the weather. The other set of properties the component has is called the state. A component's state is internal to the component and it can change when the props changing or the user interacting with the component. For example, if the weather went from sunny to raining, you would put on a raincoat and wellies. In React component terms, your state could go from "wearing a jacket" to "wearing a raincoat and wellies". You know to put on a raincoat when it's raining because you've been told by your parents. A React component would have to be told what to do by the programmer - that's you! :)
```

Comment:
But in that example, it is a computed value, I think? From your [reference reading](http://busypeoples.github.io/post/react-component-lifecycle/): 

> **componentWillReceiveProps** enables to update the state depending on the existing and upcoming props, without triggering another rendering

The official [Facebook article](https://facebook.github.io/react/docs/component-specs.html#updating-componentwillreceiveprops) has an example where state is updated in response to props changing.

In the bit above, I meant something like:
```
componentWillReceiveProps: function(nextProps) {
  this.setState({
    clothes: nextProps.weather === 'raining' ? 'jacket' : 'raincoat'
  });
}
```
But I didn't want to put the actual code in until next modules as not to be too technical from the onset.

Do you still think I should remove it?